log/formatter/general_test.go

```diff
@@ -0,0 +1,209 @@
+package formatter
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/suite"
+
+	configmock "github.com/goravel/framework/contracts/config/mocks"
+)
+
+type GeneralFormatterTestSuite struct {
+	suite.Suite
+	mockConfig  *configmock.Config
+	entry       *logrus.Entry
+	stackTraces any
+}
+
+func TestGeneralFormatterTestSuite(t *testing.T) {
+	suite.Run(t, new(GeneralFormatterTestSuite))
+}
+
+func (s *GeneralFormatterTestSuite) SetupTest() {
+	s.mockConfig = &configmock.Config{}
+	s.entry = &logrus.Entry{
+		Level:   logrus.InfoLevel,
+		Message: "Test Message",
+	}
+
+	s.mockConfig.On("GetString", "app.timezone").Return("UTC")
+	s.mockConfig.On("GetString", "app.env").Return("test")
+}
+
+func (s *GeneralFormatterTestSuite) TestGeneral_Format() {
+	general := NewGeneral(s.mockConfig)
+	formatLog, err := general.Format(s.entry)
+	s.Nil(err)
+	s.NotNil(formatLog)
+}
+
+func (s *GeneralFormatterTestSuite) TestFormatData() {
```

Comment:
The `formatData` method is a independent method, you don't need to set the test case to `GeneralFormatterTestSuite`.

 ```suggestion
func TestFormatData() {
```